its/ruling/src/test/resources/jdk6/squid-S4973.json

```diff
@@ -0,0 +1,151 @@
+{
+'jdk6:java/awt/BorderLayout.java':[
+836,
```

Comment:
I guess all these issues are going to disappear from other rule once updated to not overlap? Did you check?